plugins/modules/junos_static_routes.py

```diff
@@ -246,7 +346,92 @@
 #     route 192.168.16.0/24 next-hop 172.16.1.2;
 # }
 
+# Using gathered to gather static route facts from the device
+# Before state
+# ------------
+# admin# show routing-options
+# static {
+#     route 192.168.16.0/24 next-hop 172.16.1.2;
+#     route 192.168.47.0/24 next-hop 10.200.16.2;
+# }
+- name: Gather static routes facts from the device using junos_static_routes module
+  junipernetworks.junos.junos_static_routes:
+    state: gathered
+
+# Task output:
+# ------------
+# gathered:
+#   - address_families:
+#       - afi: ipv4
+#         routes:
+#           - dest: 192.168.16.0/24
+#             next_hop:
+#               - forward_router_address: 172.16.1.2
+#           - dest: 192.168.47.0/24
+#             next_hop:
+#               - forward_router_address: 10.200.16.2
+
+# Using rendered
+
+- name: Render platform specific commands (without connecting to the device)
+      junipernetworks.junos.junos_static_routes:
+        config:
+        - address_families:
+          - afi: ipv4
+            routes:
+            - dest: 192.168.16.0/24
+              next_hop:
+              - forward_router_address: 172.16.1.2
+        state: rendered
+
+# Task output:
+# ------------
+# rendered:
+#   - '<nc:routing-options
+#     xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0"><nc:static><nc:route><nc:name>192.168.16.0/24</nc:name>
+#     <nc:next-hop>172.16.1.2</nc:next-hop></nc:route></nc:static></nc:routing-options>'
 
+# Using parsed
+
+# parsed.cfg
+# ------------
+# <?xml version="1.0" encoding="UTF-8"?>
+# <rpc-reply message-id="urn:uuid:0cadb4e8-5bba-47f4-986e-72906227007f">
+#     <configuration changed-seconds="1590139550" changed-localtime="2020-05-22 09:25:50 UTC">
+#         <version>18.4R1-S2.4</version>
+#         <routing-options>
+#         <static>
+#             <route>
+#                 <name>192.168.16.0/24</name>
+#                 <next-hop>172.16.1.2</next-hop>
+#                 <next-hop>172.16.1.3</next-hop>
+#             </route>
+#             <route>
+#                 <name>192.168.47.0/24</name>
+#                 <next-hop>10.200.16.2</next-hop>
+#             </route>
+#         </static>
+#     </routing-options>
+#     </configuration>
+# </rpc-reply>
+
+- name: Parsed running config (without connecting to the device)
+      junipernetworks.junos.junos_static_routes:
+        running_config: "{{ lookup('file', 'parsed.cfg') }}"
+        state: parsed
+
+# Task output:
+# ------------
+# parsed:
+#   - address_families:
+#       - afi: ipv4
+#         routes:
+#           - dest: 192.168.16.0/24
+#             next_hop:
+#               - forward_router_address: '[''172.16.1.2'', ''172.16.1.3'']'
+#           - dest: 192.168.47.0/24
+#             next_hop:
+#               - forward_router_address: 10.200.16.2
 """
 RETURN = """
```

Comment:
could you also include documentation as below:  
rendered:
  description: The provided configuration in the task rendered in device-native format (offline).
  returned: when I(state) is C(rendered)
  type: list
  sample:
    - <nc:protocols xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0">
gathered:
  description: Facts about the network resource gathered from the remote device as structured data.
  returned: when I(state) is C(gathered)
  type: list
  sample: >
    This output will always be in the same format as the
    module argspec.
parsed:
  description: The device native config provided in I(running_config) option parsed into structured data as per module argspec.
  returned: when I(state) is C(parsed)
  type: list
  sample: >
    This output will always be in the same format as the
    module argspec.